lib/survey_gizmo/rest_response.rb

```diff
@@ -48,12 +48,20 @@ def initialize(rest_response)
 
   # The parsed JSON data of the response
   def data
-    @_data ||= @parsed_response['data']
+    @parsed_response['data']
   end
 
   # The error message if there is one
   def message
-    @_message ||= @parsed_response['message']
+    @parsed_response['message']
```

Comment:
memoization in this case is basically useless - there's no operation being performed except a hash lookup and hash lookups are computationally the fastest thing you can do.